Kickstarter-iOS/Views/CommentCellHeaderStackView.swift

```diff
@@ -0,0 +1,132 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+internal final class CommentCellHeaderStackView: UIStackView {
+  // MARK: - Properties
+
+  private lazy var authorNameLabel: UILabel = { UILabel(frame: .zero) }()
+
+  /// Hidden as there's need for clarification on how Comment.AuthorBadge works on GraphQL
+  /// There's currently a bug on rotation of device where the style for the authorBadgeLabel resets to a wrong state
```

Comment:
This is happening because the style is being emitted from the view model. We either need to call an input on the view model when rotation occurs so that it emits again or keep the styles in the cell and test them with snapshots. I think it's best to just rely on our snapshots here. Let's fix this before merging this PR because it may cause confusion later on.

It seems that a user can have multiple badges, this wasn't really catered for in the designs so we should just confirm what the priority of badges are (creator -> superbacker) etc.